src/adhocracy_frontend/adhocracy_frontend/static/js/Packages/ResourceActions/ResourceActions.ts

```diff
@@ -62,24 +63,6 @@ export var shareActionDirective = () => {
     };
 };
 
-export var deleteActionDirective = () => {
-    return {
-        restrict: "E",
-        template: "<a class=\"{{class}}\" href=\"\" data-ng-click=\"delete();\">{{ 'TR__DELETE' | translate }}</a>",
-        require: "^adhMovingColumn",
-        scope: {
-            resourcePath: "@",
-            parentPath: "=?",
-            class: "@"
-        },
-        link: (scope, element, attrs, column : AdhMovingColumns.MovingColumnController) => {
-            scope.delete = () => {
-                column.$broadcast("triggerDelete", scope.resourcePath);
-            };
-        }
-    };
-};
-
```

Comment:
Because it was only used for Mercator2015, where I replaced it with `adhHideAction`.